lib/logstash/inputs/file.rb

```diff
@@ -119,6 +135,11 @@ class LogStash::Inputs::File < LogStash::Inputs::Base
   # set the new line delimiter, defaults to "\n"
   config :delimiter, :validate => :string, :default => "\n"
 
+    # set the mode, defaults to "tail"
+  # in read_to_eof mode filewatch will signal the EOF with a "\x04" control character
+  # in tail mode behaviour is as before
+  config :ignore_after, :validate => :number, :default => 24 * 60 * 60
```

Comment:
File beat calls this `ignore_older`. We talked about being consistent with filebeat to aid any migration to and from these components. 

Also, can we add a more descriptive messge (copied from filebeat) - 

"If this option is specified, file input ignores any files that were modified before the specified timespan in milliseconds. This timeout also triggers a release of resources obtained"